packages/dapp/package.json

```diff
@@ -20,7 +20,9 @@
     "@widgetbot/react-embed": "^1.8.0",
     "ethers": "^5.7.2",
     "i18next": "^23.4.4",
-    "i18next-browser-languagedetector": "^7.1.0",
+    "i18next-browser-languagedetector": "^7.0.2",
```

Comment:
Was it necessary to downgrade this package? So, should we keep this in mind when bumping our packages in order to not break anything?

Reply:
@dennyscode, nice catch. No, this shouldn't be downgraded 🙂